src/OpenRiaServices.Tools/Test/CreateRiaClientFilesTaskTests.cs

```diff
@@ -438,6 +484,10 @@ public void CreateRiaClientFiles_Validate_Generated_Files(OpenRiaSharedFilesMode
             }
         }
 
+#if !NETFRAMEWORK
+        [Ignore("Do no work in NET6")]
+        [TestCategory("NET6_ERRORS")]
```

Comment:
```suggestion
```

Maybe update ignore description.

1. NET6 cannot generate code for .NET framework application so this test will never succed
2. However maybe we should add a similar test where we have a server project with connection string in "app.confg" (and where the connection string is NOT in the test projects app.config)
 * The goal would be to verify that connection strings from the server project is picked up and used during code gen (instead of codegen failing due to missing connection string)

Reply:
not a blocker: add a task  for this